tests/test_files/initialization_inference/never_init.stderr

```diff
@@ -0,0 +1,19 @@
+error: access to uninitialized path
+- initialization_inference/never_init:13:7
+13 |   take(x)
+   |        ^
+error: Unresolved variable
+- initialization_inference/never_init:9:2
+9 |   let x
+
+  |   ^^^^^^
+error: Unresolved variable
+- initialization_inference/never_init:9:2
+9 |   let x
+
+  |   ^^^^^^
+error: Unresolved variable
+- initialization_inference/never_init:9:2
+9 |   let x
+
+  |   ^^^^^^
```

Comment:
Do we want to give the error message three times?

(I guess it might be fine for now, if we later remove redundant error messages)